data-loader/core/src/test/java/com/scalar/db/dataloader/core/dataexport/ExportReportTest.java

```diff
@@ -0,0 +1,21 @@
+package com.scalar.db.dataloader.core.dataexport;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class ExportReportTest {
```

Comment:
```suggestion
class ExportReportTest {
```

Reply:
I have updated it as suggested.
Thank you.

Reply:
It looks like it's not updated?

Reply:
@brfrn169 
My apologies, I think I may have missed to commit the file. I have pushed the change now.